release-notes/release-notes58.md

```diff
@@ -188,14 +188,21 @@ This section describes issues that users of {{site.productname}} 5.8 may encount
 
 **Outline**
 
-- [](#)
+- [Unexpected behavior when moving the caret using the PageDown key](#unexpectedbehaviorwhenmovingthecaretusingthepagedownkey)
+- [Accessibility when using keyboard navigation to resize the editor](#accessibilitywhenusingkeyboardnavigationtoresizetheeditor)
 
-###
+### Unexpected behavior when moving the caret using the PageUp, PageDown, and End keys
```

Comment:
Not sure where `PageUp` came from here? The issue only applies to `PageDown` and `End` either way, so we'll need that updated.

Reply:
Whoops